src/omnicore/doc/release-notes.md

```diff
@@ -1 +1,528 @@
+- `walletcache`: Log additional information about the wallet cache and hits/misses
+- `consensus_hash`: Log additional information about each data point added to the consensus hash
+- `consensus_hash_every_block`: Generate and log a consensus hash for every block processed
+- `alerts`: Log additional information about alert transactions
+
+Special keywords:
+
+- `all`: Enable all logging categories (note: extremely verbose)
+- `none`: Disable all logging categories
+
+Usage examples:
+```
+omnicored -omnidebug=all
+```
+```
+omnicored -omnidebug=parser -omnidebug=parser_data -omnidebug=vin -omnidebug=script
+```
+
+Configuration options
+---------------------
+
+For a listing of other Omni Core specific configuration options see:
+
+- https://github.com/OmniLayer/omnicore/blob/omnicore-0.0.10/src/omnicore/doc/configuration.md
+
+Various performance improvements
+--------------------------------
+
+Omni Core was improved on several levels to increase performance and responsiveness.
+
+Previously RPC requests were blocking, creating a bottleneck for the JSON-RPC API and the whole application. Due to much finer and more targeted locking of critical sections, threads are now only blocked when they need to be, which allows concurrent requests without immediately creating a queue. The number of threads designated to handle RPC requests can be configured with the option `-rpcthreads=<n threads>`, which is set to `4` threads as per default.
+
+The serialization of token related database entries was previously JSON based, and the time to retrieve information about transactions, tokens or crowdsales, for example with [omni_gettransaction](https://github.com/OmniLayer/omnicore/blob/omnicore-0.0.10/src/omnicore/doc/rpc-api.md#omni_gettransaction), [omni_getproperty](https://github.com/OmniLayer/omnicore/blob/omnicore-0.0.10/src/omnicore/doc/rpc-api.md#omni_getproperty), [omni_getcrowdsale](https://github.com/OmniLayer/omnicore/blob/omnicore-0.0.10/src/omnicore/doc/rpc-api.md#omni_getcrowdsale) or [omni_getactivecrowdsales](https://github.com/OmniLayer/omnicore/blob/omnicore-0.0.10/src/omnicore/doc/rpc-api.md#omni_getactivecrowdsales), was primarily delayed by slow conversions of the data stored in the database. Entries are now stored in their raw byte representation, which allows much faster deserialization of the data. Benchmarks have shown this update provides a speed improvement of about 8x or more for these calls.
+
+Most notably on Microsoft Windows, many updates of the GUI, for example when connecting a new block, created the impression of a non-responsive user interface, because the client couldn't keep up with the updates. This was optimized such that no queue of update events is created in the first place, and new update requests are dropped, in case there are already events which haven't been handled.
+
```

Comment:
Hmm.. well, I wouldn't highlight that it's still not perfect, but rather that we made some improvements. Not sure how to phrase it though.

> Altogether this should provide a better user experience and allow to use Omni Core in environments with high load and demanding performance requirements.

... is probably too optimistic?